### PR TITLE
Do not mention missing `PartialOrd` impl when involving uncalled fns

### PR DESCRIPTION
**Pull request by estebank**

### CODE REVIEW

**Review by davidtwco — CHANGES_REQUESTED** [review]
LGTM, one minor comment.

File: src/librustc_typeck/check/op.rs [hunk 1]
```diff
@@ -437,7 +438,7 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
         other_ty: Ty<'tcx>,
         op: hir::BinOp,
         is_assign: IsAssign,
-    ) {
+    ) -> bool {
```
[the remark this answers is not in the document]
Comment (estebank) [reply src/librustc_typeck/check/op.rs]: Done

**Pull request merged**